api/app/controllers/spree/api/v2/storefront/countries_controller.rb

```diff
@@ -46,6 +46,15 @@ def dependencies
           end
 
           def scope
+            return shippable_countries if params['filter'['shippable']] == 'true'
+            all_countries
+          end
+
+          def shippable_countries
+            Spree::ShippingMethod.all.map(&:zones).flatten.reduce([]) { |countries, zone| countries + zone.country_list }.uniq
```

Comment:
`Spree::ShippingMethod.where(display_on: ['both', 'front_end'])` we only want to display countries attached to shipping methods visible on frontend